README.md

```diff
@@ -95,26 +95,45 @@ envs
 
 The configuration in the `teams` subdirectories represent the individual teams that are running workloads in our EKS clusters.
 
-Each `team` subdirectory in turn has environment subdirectories. The configuration in those directories are Kubernetes manifests that represent the individual workload that is running for each team and in each environment.
+Each `team` subdirectory in turn has environment subdirectories. The configuration in those directories are Kubernetes manifests that represent the individual workload that is running for each team and in each environment. The `team-spock` implements the CDK EKS Blueprints pattern [GitOps based Multi-cluster add-on and Apps Management using Crossplane and ArgoCD](https://github.com/aws-samples/cdk-eks-blueprints-patterns/blob/main/docs/patterns/crossplane-argocd-gitops.md) 
+
 
 **NOTE** Some team environment directories may contain additional `yaml` files such as an `ingress.yaml`
 
 ```
 teams
 ├── team-burnham
-│   ├── dev
-│   │   ├── deployment.yaml
-│   │   └── service.yaml
-│   ├── prod
-│   │   ├── deployment.yaml
-│   │   └── service.yaml
-│   └── test
-│       ├── deployment.yaml
-│       └── service.yaml
+│   ├── dev
+│   │   ├── Chart.yaml
+│   │   ├── templates
+│   │   │   ├── deployment.yaml
+│   │   │   ├── service.yaml
+│   │   └── values.yaml
+│   ├── prod
+│   │   ├── Chart.yaml
+│   │   ├── templates
+│   │   │   ├── deployment.yaml
+│   │   │   ├── service.yaml
+│   │   └── values.yaml
+│   ├── test
+│   │   ├── Chart.yaml
+│   │   ├── templates
+│   │   │   ├── deployment.yaml
+│   │   │   ├── service.yaml
+│   │   └── values.yaml
+├── team-spock
```

Comment:
can we have an explanation of what this team does? Like team-spock is a platform team, etc and it support the XYZ pattern from the patterns repo.

Reply:
done. updated README with team-spock description and a link to the patterns repo